app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/AdditionalFields.tsx

```diff
@@ -44,6 +46,11 @@ export class AdditionalFields extends React.Component<IRollingRedBlackStrategyAd
     this.forceUpdate();
   };
 
+  private handleChange = (key: string, value: any) => {
```

Comment:
did typing `value` as a string blow up on you?

Reply:
🤷‍♀️ This was totally c&p from the redblack AdditionalFields.

I'll give it a whirl.

Reply:
Gave it the `value: string` treatment